commons/deployment/src/main/resources/dev-ui/qwc-quarkus-hilla-endpoints.js

```diff
@@ -0,0 +1,200 @@
+import {css, html, LitElement} from 'lit';
+import {columnBodyRenderer} from '@vaadin/grid/lit.js';
+import {hillaEndpoints as endpoints} from 'build-time-data';
+import '@vaadin/grid';
+import '@vaadin/grid/vaadin-grid-sort-column.js';
+import '@vaadin/grid/vaadin-grid-tree-column.js';
+import '@vaadin/tooltip/src/vaadin-tooltip.js';
+import '@vaadin/icon';
+
+/**
+ * This component shows the Rest Easy Reactive Parameter Converter Providers
+ */
+export class QwcQuarkusHillaEndpoints extends LitElement {
+
+    static styles = css`
+        .datatable {
+            height: 100%;
+            padding-bottom: 10px;
+            border: none;
+        }
+        
+        vaadin-grid-tree-toggle {
+            width: 100%;
+        }
+
+        .annotation {
+            color: var(--lumo-contrast-50pct);
+        }
+
+        .primary {
+            color: var(--lumo-primary-text-color);
+        }
+
+        .parameters {
+            > ::after {
+                content: ",";
+            }
+
+            :last-child::after {
+                content: "";
+            }
+        }
+
+        .PermitAll {
+            color: var(--lumo-success-text-color);
+        }
+
+        .RolesAllowed {
+            color: var(--lumo-success-text-color);
+        }
+
+        .AnonymousAllowed {
+            color: var(--lumo-warning-text-color);
+        }
+
+        .DenyAll {
+            color: var(--lumo-error-text-color);
+        }
+    `;
+
+    static properties = {
+        _endpoints: {state: true}, _expandedItems: {state: true}
+    };
+
+    constructor() {
+        super();
+        this._endpoints = endpoints;
+        this._expandedItems = endpoints;
+    }
+
+    render() {
+        if (this._endpoints) {
+            return this._renderEndpointGrid()
+        } else {
+            return html`No beans found`;
+        }
+    }
+
+    _renderEndpointGrid() {
+        if (this._endpoints) {
+            return html`
+                <vaadin-grid .dataProvider="${this._dataProvider}" .expandedItems="${this._expandedItems}"
+                             class="datatable" theme="row-stripes">
+                    <vaadin-grid-sort-column header="Class"
```

Comment:
Sorting does not seem to work. Would it make sense to remove it?

Reply:
That's true, didn't know how to correctly specify the sort parameter. If you can give me a hint, would be nice, otherwise we could remove it.